### ISSUE TITLE
Add variant for " / ' (double quote / single quote) characters

### ISSUE THREAD

**Issue by jakubdeka-118118money** [tonsky/FiraCode#1357]
A stylistic option for the " / ' making them bigger and bolder would be good and could imrpove readability when using small size.

I have been using Input font for ages and the quote characters are a lot more readable

![image](https://user-images.githubusercontent.com/44904871/150963951-49e55c2e-6d2e-452e-96dc-53ceaffd4921.png)

vs

![image](https://user-images.githubusercontent.com/44904871/150963824-93bcc77d-51a8-477b-8a32-04205d89ca13.png)

**Comment by Thyrisis** [comment]
I agree this would be a useful addition - default single quote in particular I find relatively small

**Comment by tonsky** [comment]
Good idea, thanks!

**Comment by nmsmith** [comment]
IMO it's worth considering making larger quotes the default, because I doubt many people would prefer the tiny quotes. In general, bigger and chunkier glyphs are easier to read.
